app/models/abstract_contributor.rb

```diff
@@ -7,8 +7,8 @@ class AbstractContributor < ApplicationRecord
   SEPARATOR = '|'
 
   PERSON_ROLES = [
-    'Advisor',
     'Author',
```

Comment:
What would you think about dropping a comment here that we expect `Author` to **remain** the first element in this array?

Reply:
Sounds good - comment added.

Reply:
@peetucket Thx! Feel free to merge when 💚